src/Objectivity.AutoFixture.XUnit2.AutoMoq/Attributes/AutoMoqDataAttribute.cs

```diff
@@ -15,16 +15,18 @@ namespace Objectivity.AutoFixture.XUnit2.AutoMoq.Attributes
     using Ploeh.AutoFixture;
     using Ploeh.AutoFixture.AutoMoq;
     using Ploeh.AutoFixture.Xunit2;
+    using SpecimenBuilder;
 
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1019:DefineAccessorsForAttributeArguments", Justification = "MJI: There is no need to provide public accessor for ignoreVirtualMembers property")]
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
     public sealed class AutoMoqDataAttribute : AutoDataAttribute
     {
-        public AutoMoqDataAttribute()
-            : this(new Fixture())
+        public AutoMoqDataAttribute(bool ignoreVirtualMembers = false)
+            : this(new Fixture(), ignoreVirtualMembers)
         {
         }
 
-        public AutoMoqDataAttribute(IFixture fixture)
+        public AutoMoqDataAttribute(IFixture fixture, bool ignoreVirtualMembers = false)
```

Comment:
Ok for MemberAutoMoqDataAttribute. Other cases I planned to discuss with you but I can see that now there is nothing to discuss :).